libs/pubsub/query/query.go

```diff
@@ -195,6 +197,28 @@ func (q *Query) Matches(events map[string][]string) bool {
 			op = OpEqual
 		case rulecontains:
 			op = OpContains
+		case ruleexists:
+			op = OpExists
+			if strings.Contains(tag, ".") {
+				// Searching for a full "type.attribute" event.
+				_, ok := events[tag]
+				if !ok {
+					return false
+				}
+			} else {
+				foundEvent := false
+
+			loop:
+				for k := range events {
```

Comment:
```suggestion
				for eventType := range events {
```

Reply:
I can see the point of picking a better variable name than just `k`. `eventType` is not the correct term, I believe. Perhaps a better name would be `compositeTag`, which seems to be the term used in `event_bus.go` line 124.

Reply:
Changed variable name to `compositeTag`.

Reply:
eventType is the correct term

Reply:
Are you certain? The `events` slice will contain entries of the form `<eventType>.<eventAttribute>` as outlined in the `validateAndStringifyEvents` function of the event bus.

Reply:
oh, so it's a composite key (`type.attribute`). then `compositeTag` or better `compositeKey` make total sense!